app/assets/javascripts/dashboard/advanced_dataset/dataset_action_view.js

```diff
@@ -56,12 +56,8 @@ class DatasetActionView extends React.PureComponent<Props, State> {
 
     const volumeTracingMenu = (
       <Dropdown overlay={menu} trigger={["click"]}>
-        <a href="#" title="Create Volume Tracing">
-          <img
-            src="/assets/images/volume.svg"
-            alt="volume icon"
-            style={centerBackgroundImageStyle}
-          />{" "}
+        <a href="#" title="Create Volume tracing">
```

Comment:
You made this a small `t` by accident.
```suggestion
        <a href="#" title="Create Volume Tracing">
```
You'll need to recreate the snapshots again then, unfortunately ^.~